### ISSUE TITLE
Use pnpm for better package management

### ISSUE THREAD

**Issue by sdilonedev** [PrintfDead/discord-bot#1]
Well I see that you are using NPM as your package manager, it's good, but pnpm is the fastest and will probably be a standard in Node.

Here is the comparison link: https://pnpm.io/es/feature-comparison

**Comment by jjprintf** [comment]
Yes, as well as yarn, but this code is to be used as a bot base, you can change the package manager you want to use yourself

**Comment by nnachonesa** [comment]
How i can see, you are so professional. 🍷
